custom_components/artnet_led/light.py

```diff
@@ -87,14 +90,21 @@ async def async_setup_platform(hass: HomeAssistant, config, async_add_devices, d
     host = config.get(CONF_NODE_HOST)
     port = config.get(CONF_NODE_PORT)
 
+    real_host = config.get(CONF_NODE_HOST_OVERRIDE)
+    if len(real_host) == 0 :
+        real_host = host
+    real_port = config.get(CONF_NODE_PORT_OVERRIDE)
+    if real_port == None:
+        real_port = port
```

Comment:
Possible alternative could be:

`real_port = config.get(CONF_NODE_PORT_OVERRIDE, port)`